### PR TITLE
Closes #773: Migration for D7 headed text paragraph type.

### LINKED ISSUE #773: Migrate Text Area paragraph type
See migration mapping doc for details: https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0

### PR DESCRIPTION
**Pull request by smitahantzios**
Closes #773 
This is to migrate the Paragraph type - uaqs_content_chunks_headed_text from D7 to D9

## Description
This is to migrate the Paragraph type - uaqs_content_chunks_headed_text from D7 to D9

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_headed_text.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+id: az_paragraph_headed_text
+label: AZ Pragrapgh Headed Text
```
Comment (camikazegreen) [inline modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_headed_text.yml:2]:
```suggestion
label: AZ Paragraph Headed Text
```

**Comment by danahertzberg** [comment]
Looks like this build got reaped too. Before I rebuild it, is there any way to compare content in these migrations before the Page content type migration is complete?

**Comment by camikazegreen** [comment]
@danahertzberg You can turn on the Paragraphs Admin module to verify that they are there, but what you can see is going to be relatively limited until we migrate the pages.

**Comment by danahertzberg** [comment]
> @danahertzberg You can turn on the Paragraphs Admin module to verify that they are there, but what you can see is going to be relatively limited until we migrate the pages.

@camikazegreen Just verify the paragraph types are there? I'm asking if there's any way to compare content from the source to the destination at this point.

**Comment by danahertzberg** [comment]
Ready to merge after security update

File: modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_headed_text.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+process:
+
+  field_az_text_area:
+    plugin: sub_process
+    source: field_uaqs_summary
+    process:
+      delta: delta
+      value: value
+      summary: summary
```
Comment (tadean) [inline modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_headed_text.yml:24]: The `summary` field is not available on `field_az_text_area`

Comment (smitahantzios) [reply modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_headed_text.yml]: `summary` field is removed.

**Comment by joeparsons** [comment]
@smitahantzios Could you merge the main branch into this one as well?

**Comment by smitahantzios** [comment]
Merged main branch to this branch.

**Review by tadean — APPROVED** [review]

**Pull request merged**